docs/user_guides/tls_proxy.md

```diff
@@ -19,11 +19,8 @@ sudo apt install openssl
 mkdir cert && cd cert
 touch openssl.cnf gen.sh
 ```
-Copy the configuration below into the file. You can modify gen.ssh configurations.
+Copy the configuration below into the file.
 
-CommonName is a required configuration. The client needs to specify the server name when connecting. The server name is common name.
```

Comment:
Configuration of `CommonName` is required. `CommonName` refers to the server name that the client needs to specify when connecting.